app/src/main/java/io/appium/uiautomator2/model/UiSelectorHelper.java

```diff
@@ -18,60 +18,63 @@
 
 import android.view.accessibility.AccessibilityNodeInfo;
 
+import androidx.annotation.Nullable;
 import androidx.test.uiautomator.UiSelector;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 import io.appium.uiautomator2.utils.Attribute;
 
-public class CustomUiSelector {
+public class UiSelectorHelper {
+    private static final Attribute[] MATCHING_ATTRIBUTES = new Attribute[]{
```

Comment:
Is MATCHING_ATTRIBUTES subset of SUPPORTED_ATTRIBUTES?
(I mean these attributes are part of SUPPORTED_ATTRIBUTES)

Reply:
yes they are. We only use a limited set of attributes for selector matching